tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -487,7 +487,7 @@ public static async Task<RecordEntry> CreateEntryAsync(HttpRequest request)
             byte[] bytes = await ReadAllBytes(request.Body).ConfigureAwait(false);
 
             entry.Request.Body = CompressionUtilities.DecompressBody(bytes, request.Headers);
```

Comment:
The bytes are not mutated by DecompressBody, right?

Reply:
Sure.